src/types/subscription.ts

```diff
@@ -160,25 +160,41 @@ export interface ParticipationUpdates {
  * * Add emitted ARC-28 events via `arc28Events`
  * * Balance changes in algo or assets
  */
-export type SubscribedTransaction = TransactionResult & {
+export class SubscribedTransaction extends algosdk.indexerModels.Transaction {
   /** The transaction ID of the parent of this transaction (if it's an inner transaction). */
   parentTransactionId?: string
```

Comment:
We need to review this since as I noted above we actually used root not parent in the implementation